### ISSUE TITLE
There is no way to approve identity requests

### ISSUE THREAD

**Issue by mrdigital0** [ossified/openCad#39]
Pressing Accept doesn't do anything. 

theres no php call for the button link.

**Comment by ossified** [comment]
Yeah, unfortunately I never got to finishing the backend of the Identity Management system. This is an open bug that I unfortunately have no timeline for resolution.
